localstack/services/cloudwatch/provider_v2.py

```diff
@@ -214,3 +221,46 @@ def tag_resource(
     ) -> TagResourceOutput:
         self.tags.tag_resource(resource_arn, tags)
         return TagResourceOutput()
+
+    def put_dashboard(
+        self, context: RequestContext, dashboard_name: DashboardName, dashboard_body: DashboardBody
+    ) -> PutDashboardOutput:
+        store = self.get_store(context.account_id, context.region)
+        store.Dashboards[dashboard_name] = dashboard_body
+        return PutDashboardOutput()
+
+    def get_dashboard(
+        self, context: RequestContext, dashboard_name: DashboardName
+    ) -> GetDashboardOutput:
+        store = self.get_store(context.account_id, context.region)
+        dashboard_body = store.Dashboards.get(dashboard_name)
+        dashboard_arn = arns.cloudwatch_dashboard_arn(
+            dashboard_name, account_id=context.account_id, region_name=context.region
+        )
+        if not dashboard_body:
+            raise InvalidParameterValueException(f"Dashboard {dashboard_name} does not exist.")
+        return GetDashboardOutput(
+            DashboardName=dashboard_name, DashboardBody=dashboard_body, DashboardArn=dashboard_arn
+        )
+
+    def delete_dashboards(self, context: RequestContext, dashboard_names: DashboardNames) -> None:
+        store = self.get_store(context.account_id, context.region)
```

Comment:
NIT: does it throw an error when we try to delete a dashboard that does not exist?

Reply:
No. it doesn't